main.py

```diff
@@ -19,34 +21,45 @@ def slack_events_url_endpoint(request):
     return slack_events_receive_callback(request)
 
 
+credentials_path = f"{os.path.abspath('__file__')}/app-credentials.json"
+
+
 def slack_events_receive_callback(request) -> dict:
     headers = dict(request.headers)
-    data = {}
+    response = {}
 
     if not is_request_timestamp_valid(headers["X-Slack-Request-Timestamp"]):
-        data["status"] = "failed"
-        data["message"] = "The request timestamp has expired."
+        response["status"] = "failed"
+        response["message"] = "The request timestamp has expired."
 
     request_data = request.get_json(force=True)
 
     request_type = request_data.get("type")
 
-    event_handler = SlackEventApiHandler()
-
     if request_type == "url_verification":
-        data = event_handler.respond_to_url_verification(request_data)
+        response = SlackEventApiHandler.respond_to_url_verification(request_data)
```

Comment:
`respond_to_url_verification` now changed to class method as it does not need to use other class methods to work